ibis/backends/mssql/registry.py

```diff
@@ -112,6 +116,39 @@ def _temporal_delta(t, op):
     return sa.func.datediff(sa.literal_column(op.part.value.upper()), right, left)
 
 
+def _literal(t, op):

```

Comment:
I'm going to factor this out into a separate PR, these changes should be unrelated to odbc.

Reply:
Nevermind, turns out these actually require a bunch of additional munging.